Makefile.toml

```
@@ -47,6 +47,12 @@ description = "Build Code with preview features"
 command = "cargo"
 args = ["build", "--verbose", "--release", "--package=aranya-daemon", "--features=preview", "${@}"]
 
+[tasks.build-daemon]
```

To follow the naming convention since preview and experimental flags are enabled:
```suggestion
[tasks.build-daemon-experimental]
```